src/Routes/ParticipantRoutes.tsx

```diff
@@ -63,6 +63,78 @@ export default [
     exact
     key="nhspreregistration"
   />,
+  <Route
+    path="/nhsenwl"
+    render={() => {
+      ReactGA.pageview(
+        "/participants/introduction?utm_source=poster&utm_medium=print&utm_campaign=nhsenwl"
+      );
+      return <NhsPreRegistration />;
+    }}
+    strict
+    exact
+    key="nhsenwl"
+  />,
+  <Route
+    path="/nhseswl"
+    render={() => {
+      ReactGA.pageview(
+        "/participants/introduction?utm_source=poster&utm_medium=print&utm_campaign=nhseswl"
+      );
+      return <NhsPreRegistration />;
+    }}
+    strict
+    exact
+    key="nhseswl"
+  />,
+  <Route
+    path="/nhsegp"
+    render={() => {
+      ReactGA.pageview(
+        "/participants/introduction?utm_source=poster&utm_medium=print&utm_campaign=nhsegp"
+      );
+      return <NhsPreRegistration />;
+    }}
+    strict
+    exact
+    key="nhsegp"
+  />,
+  <Route
+    path="/nhsegptext1"
+    render={() => {
+      ReactGA.pageview(
+        "/participants/introduction?utm_source=sms&utm_medium=referral&utm_campaign=nhsegptext1"
+      );
+      return <NhsPreRegistration />;
+    }}
+    strict
+    exact
+    key="nhsegptext1"
+  />,
+  <Route
+    path="/nhsegptext2"
+    render={() => {
+      ReactGA.pageview(
+        "/participants/introduction?utm_source=sms&utm_medium=referral&utm_campaign=nhsegptext2"
+      );
+      return <NhsPreRegistration />;
+    }}
+    strict
+    exact
+    key="nhsegptext2"
+  />,
+  <Route
+    path="/nhsegptext3"
+    render={() => {
+      ReactGA.pageview(
+        "/participants/introduction?utm_source=sms&utm_medium=referral&utm_campaign=nhsegptext3"
+      );
+      return <NhsPreRegistration />;
+    }}
+    strict
+    exact
+    key="nhsegptext3"
+  />,
```

Comment:
could we refactor for reusability of code?
e.g. 
```
const getRoute = (path,) => {
  return (
    <Route
      path={path}
      ...
    />
  );
};
```

```
const routes = [
  getRoute('/path1', ... ),
  getRoute('/path2', ... ),
  ...
]
```

Reply:
If I was going to do this I would map rather than multiple function calls but given the time sensitivity/other meetings today I couldn't refactor the component.  Definitely one for improvements though

Reply:
Totally take your point though and it is a very good one, just introduces too much risk with such little time.  Currently there is no risk with this change as adding these routes will not affect anything, refactoring will.  I would do something like this:

`export default routes.map(({ path, component, isProtected, gaPageView, key }) => {
  const RouteComponent = isProtected ? ProtectedRoute : Route;

  return (
    <RouteComponent
      path={path}
      exact={true}
      strict={true}
      key={key}
      render={(props) => {
        if (gaPageView) {
          ReactGA.pageview(gaPageView);
        }
        return React.createElement(component, props);
      }}
    />
  );
});`

and have an array of route configurations which would negate the need for `exact` and `strict` in each route.  Although a case could be made NOT to do this as these are removed as mandatory in `react-router v6` so would the time be better used converting the application to use the new router(as there are other added benefits)

Reply:
could actually go one stage further and memoise the Route render for better performance